packages/cli/bin/cli.js

```diff
@@ -110,9 +111,13 @@ const performChecks = argv => {
   }
 };
 
+const setRequestHeaders = () => {
+  addRequestHeader('X-Parent-User-Agent', `HubSpot CLI/${pkg.version}`);
```

Comment:
Is this a standard header?

Can this be `HubSpot CLI-lib/{version}`?

Reply:
Just to close the loop here, I updated this to use the existing `User-Agent` header (see my latest comment)